pkg/cmd/dedicated/deregister/deregister.go

```diff
@@ -54,6 +56,8 @@ func NewDeRegisterClusterCommand(f *factory.Factory) *cobra.Command {
 	flags.StringVar(&opts.accessToken, "access-token", "", f.Localizer.MustLocalize("dedicated.deregistercluster.flag.accessToken.description"))
 	flags.StringVar(&opts.selectedClusterId, "cluster-id", "", f.Localizer.MustLocalize("dedicated.deregisterCluster.flag.clusterId.description"))
 
+	_ = flags.MarkHidden("cluster-mgmt-api-url")
```

Comment:
Can you refactor those 2 lines into a function?
It will make it easier to be removed, I hope 🙂